src/test/kotlin/com/terraformation/backend/accelerator/db/ReportMetricStoreTest.kt

```diff
@@ -78,74 +84,212 @@ class ReportMetricStoreTest : DatabaseTest(), RunsAsDatabaseUser {
+                description = "Project objectives metric description",
+                name = "Project Objectives Metric",
+                reference = "3.0",
+                type = MetricType.Impact,
+            )
 
-      assertEquals(
-          listOf(
-              // Ordered by reference then ID
-              ExistingStandardMetricModel(
-                  id = standardMetricId1,
-                  component = MetricComponent.Climate,
-                  description = "Climate standard metric description",
-                  name = "Climate Standard Metric",
-                  reference = "3.0",
-                  type = MetricType.Activity,
-              ),
-              ExistingStandardMetricModel(
-                  id = standardMetricId3,
-                  component = MetricComponent.ProjectObjectives,
-                  description = "Project objectives metric description",
-                  name = "Project Objectives Metric",
-                  reference = "3.0",
-                  type = MetricType.Impact,
-              ),
-              ExistingStandardMetricModel(
-                  id = standardMetricId2,
-                  component = MetricComponent.Community,
-                  description = "Community metric description",
-                  name = "Community Metric",
-                  reference = "5.0",
-                  type = MetricType.Outcome,
-              ),
-          ),
-          store.fetchAllStandardMetrics())
+        assertEquals(
+            listOf(
+                // Ordered by reference then ID
+                ExistingStandardMetricModel(
+                    id = standardMetricId1,
+                    component = MetricComponent.Climate,
+                    description = "Climate standard metric description",
+                    name = "Climate Standard Metric",
+                    reference = "3.0",
+                    type = MetricType.Activity,
+                ),
+                ExistingStandardMetricModel(
+                    id = standardMetricId3,
+                    component = MetricComponent.ProjectObjectives,
+                    description = "Project objectives metric description",
+                    name = "Project Objectives Metric",
+                    reference = "3.0",
+                    type = MetricType.Impact,
+                ),
+                ExistingStandardMetricModel(
+                    id = standardMetricId2,
+                    component = MetricComponent.Community,
+                    description = "Community metric description",
+                    name = "Community Metric",
+                    reference = "5.0",
+                    type = MetricType.Outcome,
+                ),
+            ),
+            store.fetchAllStandardMetrics())
+      }
+
+      @Test
+      fun `throws access denied exception for non-accelerator admin`() {
+        deleteUserGlobalRole(role = GlobalRole.AcceleratorAdmin)
```

Comment:
Maybe worth testing the Read Only global role too since we've had instances of that role having the wrong privileges.